### ISSUE TITLE
What is libffi-dev? Why sudo startup.sh?

### ISSUE THREAD

**Issue by gandie** [Stakdek/zope#13]
Two-in-one issue:

**Bottom line first:** 
I do not want to run `install.sh` with `sudo` calls in it as I want to build many zope environments in parallel, making `apt-get install` calls redundant.

1. Please explain from `install.sh`:
```bash
echo "Installing system requirements"
sudo apt-get install libffi-dev
```
Why is this done in install.sh ? Why do i need this package?

I deactivated these lines and i came out perfectly fine without them.

Why pollute `install.sh` with such things?

I see no reason for this to be done here as it should be mentioned as a requirement (for what?!) in the README.md

2. Another one from `install.sh`, at the end:
```bash
echo "now running Zope with bash startup.sh."
sudo bash startup.sh
```

Why sudo?

I don't see any reason for this at all. Removed these lines as well and had no problems.

**Comment by Stakdek** [comment]
1. This is a module to ensure you can run compiled C functions which is needed to install the Products. 
If you can not do this all `pip install *` will break. 
For example on all Ubuntu Server LTS‘ will this happen until you install this module

**Comment by Stakdek** [comment]
2. I'm a little suspicious myself about the sudo. The sudo is needed if you don't want to run zope on "localhost", but on a real ip. 
I would like to leave that out, but it doesn't hurt.

**Comment by Stakdek** [comment]
What do you think? @gandie
Will be a mention in the readme better than this?

**Comment by gandie** [comment]
Correct.

Maybe u can create two chapters in README describing development and production usage. Add your description of `libffi-dev` as well - in the requirements chapter.

Additionally it would not hurt to describe how to setup `apache` and/or `nginx` in front of zope using WSGI. :heart_eyes:

**Comment by Stakdek** [comment]
Closed with PR #14
